recipes-multimedia/imx-dsp/imx-dsp_2.1.9.bb

```diff
@@ -1,15 +1,16 @@
-# Copyright 2017-2023 NXP
+# Copyright 2017-2022,2024 NXP
 
 DESCRIPTION = "i.MX DSP Wrapper, Firmware Binary, Codec Libraries"
 LICENSE = "Proprietary"
 LIC_FILES_CHKSUM = "file://COPYING;md5=10c0fda810c63b052409b15a5445671a"
 
+IMX_SRCREV_ABBREV = "a3074e2"
 
-inherit fsl-eula-unpack autotools pkgconfig
+inherit fsl-eula-recent autotools pkgconfig fsl-eula2-unpack2
```

Comment:
??

Reply:
@zelan-nxp, please don't upstream `fsl-eula-recent` and `fsl-eula2` usage. Update the recipe without these classes. I need to work on it before upstreaming.